src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -557,7 +557,7 @@
   <data name="Profile_IconBrowse.Content" xml:space="preserve">
     <value>Browse...</value>
   </data>
-  <data name="Profile_Padding.Header" xml:space="preserve">
+  <data name="Profile_Padding.Text" xml:space="preserve">

```

Comment:
We can't take this loc change -1 days before we ship. You're gonna have to figure out how to make up for it in code. Keep it `Header` and use `RS_`, file a bug to remove the workaround.

Reply:
Fixed!